src/analytics.ts

```diff
@@ -67,6 +67,7 @@ export interface InitOptions {
   plan?: Plan
   retryQueue?: boolean
   obfuscate?: boolean
+  protocol?: 'http' | 'https'
```

Comment:
Since this setting is specific to the segment.io plugin, I think it makes more sense to add this to https://github.com/segmentio/analytics-next/blob/aa423cc828fc9da0a4df6603dbda9ab3978db698/src/plugins/segmentio/index.ts#L15-L33

That way there's also no confusion on why we're still loading bundles and talking to destinations via https instead of http :smile:

Reply:
Hey @chrisradek just curious where they'd need to set that value if we move it to the deliveryStrategy property? I'm guessing somewhere in the plugin configuration for segment.io? I want to make sure I convey how their devs can use the property properly.

Reply:
I was imagining it would live next to `apiHost`, so if they wanted to talk to their dev environment they'd be able to load analytics like:
```js
analytics.load('writeKey', {
  integrations: {
    "Segment.io": {
      apiHost: "localhost",
      protocol: "http"
    }
  }
);
```

Reply:
Oh I understand now. Yes, I agree with that - sorry, the GitHub preview of your snippet centered on that `deliveryStrategy`, so i thought you meant to put it there.

Reply:
Woops, I should have called out the interface! 😬